### PR TITLE
Added endpoint for getting configured identity providers.

### PR DESCRIPTION
**Pull request by michaelvidal24**
This change is [Reviewable](https://reviewable.io/reviews/healthcatalyst/fabric.identity/48)

### CODE REVIEW

File: Fabric.Identity.API/Quickstart/Account/AccountController.cs [hunk 1]
```diff
@@ -53,6 +53,7 @@ public class AccountController : Controller
             DocumentDbUserStore documentDbUserStore,
             ILogger logger,
             IExternalIdentityProviderServiceResolver externalIdentityProviderServiceResolver,
+            AccountService account,
```
Comment (bsmithvt) [inline Fabric.Identity.API/Quickstart/Account/AccountController.cs]: Minor - can we name this `accountService` for clarity?

File: Fabric.Identity.API/Services/IdentityProviderConfigurationService.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            _httpContextAccessor = httpContextAccessor ?? throw new ArgumentNullException(nameof(httpContextAccessor));
+        }
+
+        public ICollection<ExternalProvider> GetConfiguredIdentityProviders()
+        {
+            var schemes = _httpContextAccessor.HttpContext.Authentication.GetAuthenticationSchemes();
+
+            var providers = schemes
+                .Where(x => x.DisplayName != null && !AccountOptions.WindowsAuthenticationSchemes.Contains(x.AuthenticationScheme))
```
Comment (bsmithvt) [inline Fabric.Identity.API/Services/IdentityProviderConfigurationService.cs]: Should there be a filter on whitespace/empty DisplayName?

File: Fabric.Identity.IntegrationTests/IdentityProvidersControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Fabric.Identity.IntegrationTests
+{
+    public class IdentityProvidersControllerTests : IntegrationTestsFixture
+    {
+        private readonly string _identityProvidersBaseUrl = "/api/identityProviders";
+        [Fact]
+        public async Task UsersController_Search_InvalidIdentityProvider_ReturnsBadRequest()
```
Comment (bsmithvt) [inline Fabric.Identity.IntegrationTests/IdentityProvidersControllerTests.cs]: Is the test method name accurate? It doesn't appear to match the actual test.

File: Fabric.Identity.API/Quickstart/Account/AccountController.cs (hunk 1 above)
Comment (michaelvidal24) [reply Fabric.Identity.API/Quickstart/Account/AccountController.cs]: good point, i'll make that change.

File: Fabric.Identity.API/Services/IdentityProviderConfigurationService.cs (hunk 2 above)
Comment (michaelvidal24) [reply Fabric.Identity.API/Services/IdentityProviderConfigurationService.cs]: In this case, I don't think that is necessary. We are only trying to filter out authentication schemes like Bearer which will have a null display name.

File: Fabric.Identity.IntegrationTests/IdentityProvidersControllerTests.cs (hunk 3 above)
Comment (michaelvidal24) [reply Fabric.Identity.IntegrationTests/IdentityProvidersControllerTests.cs]: Good catch - i'll fix that.

**Pull request merged**